include/fuzzuf/executor/native_linux_executor.hpp

```diff
@@ -149,5 +143,6 @@ class NativeLinuxExecutor : public Executor {
     epoll_event fork_server_read_event;
 
     bool record_stdout_and_err;
-    fuzzuf::executor::EnvironmentVariables environment_variables;
+    std::vector< std::string > environment_variables;
+    std::vector< char* > raw_environment_variables;
```

Comment:
Would you leave a comment like "raw_environment_variables == [s.c_str() for s in environment_variables]" here?

Reply:
fixed